Jobs/PrGate.yml

```diff
@@ -164,19 +167,27 @@ jobs:
           self_host_agent: true
         ${{ else }}:
           self_host_agent: false
-        extra_build_args: CODE_COVERAGE=TRUE
+        ${{ if eq(parameters.calculate_code_coverage, true) }}:
+          extra_build_args: CODE_COVERAGE=TRUE
+        ${{ else }}:
+          extra_build_args: ''
 
-- job: PublishCoverage
-  dependsOn:
-  - ${{ each item in parameters.build_matrix }}:
-    - Build_${{ item.Key }}
-  condition: and(not(Canceled()), not(Failed()))
+- ${{ if eq(parameters.calculate_code_coverage, true) }}:
+  - job: PublishCoverage

```

Comment:
Not related to the PR, but I filed an issue to try to optimize code coverage publishing in the pipeline here - https://github.com/microsoft/mu_devops/issues/267. Please let me know if you have some time to look into it.

Reply:
I've left a comment on the issue. I'd be happy to look more into it also. 

I'd like to get this in first, though, if that is acceptable.

Reply:
Yes, that's an existing problem that doesn't need to block this.